libs/langchain/langchain/chains/base.py

```diff
@@ -461,18 +481,10 @@ def prep_inputs(self, inputs: Union[Dict[str, Any], Any]) -> Dict[str, str]:
                 # If there are multiple input keys, but some get set by memory so that
                 # only one is not set, we can still figure out which key it is.
                 _input_keys = _input_keys.difference(self.memory.memory_variables)
-            if len(_input_keys) != 1:
```

Comment:
If you're able could you follow up with a PR to update the doc-string in prep_inputs since it's not doing a validation anymore?